app/views/third_spaces/new.html.erb

```diff
@@ -78,6 +78,8 @@
           </div>
         </div>
 
+        <%= hidden_field_tag 'location_json', @json %>
```

Comment:
Glad we kept this too, passing json might be sloppy, but I wasn't sure how else to pass info without making excessive API calls